client/layout/transient-notices/index.js

```diff
@@ -24,7 +30,7 @@ class TransientNotices extends Component {
 
 		return (
 			<SnackbarList
-				notices={ notices }
+				notices={ notices.concat( notices2 ) }
```

Comment:
Do the `concat` in the component instead of the `useSelect` as concat creates a new object reference.

Reply:
Nice one and good spotting. My only concern is that others are not going to pick up why this is done this way in future (unless they know the nuances of render perf in React), so I'm contemplating whether a comment would be good here maybe? I don't mind if you don't because I don't like those kind of comments a whole lot, but it may help someone.